sequencers/modeshape-sequencer-epub/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,20 @@
+# Direct log messages to stdout
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+log4j.appender.stdout.Target=System.out
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+log4j.appender.stdout.layout.ConversionPattern=%d{ABSOLUTE} %5p %m%n
+
+# Root logger option
+log4j.rootLogger=INFO, stdout
+
+# Set up the default logging to be INFO level, then override specific units
+log4j.logger.org.modeshape=INFO
+
+# Turn off the serialization protocol used in Hibernate Search (for clustering)
```

Comment:
the logging statements below this comment can be removed; ModeShape 5 doesn't use HSearch or ISPN anymore.